src/theory/fp/fp_converter.cpp

```diff
@@ -0,0 +1,54 @@
+/*********************                                                        */
+/*! \file fp_converter.cpp
+ ** \verbatim
+ ** Original author: Martin Brain
+ ** Major contributors: 
+ ** Minor contributors (to current version): 
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2014  University of Oxford
+ ** See the file COPYING in the top-level source directory for licensing
+ ** information.\endverbatim
+ **
+ ** \brief [[ Conversion of floating-point operations to bit-vectors using symfpu. ]]
+ **
+ **/
+
+
+#include "theory/fp/fp_converter.h"
+
+#include <stack>
+
+
+namespace CVC4 {
+namespace theory {
+namespace fp {
+
+
+  size_t PairTypeNodeHashFunction::operator()(const std::pair<TypeNode, TypeNode> &p) const {
```

Comment:
Please apply clang-format to the buffer. The two spaces at the beginning is off.